packages/twenty-server/src/core/open-api/open-api.service.ts

```diff
@@ -25,16 +25,22 @@ import {
   getSingleResultSuccessResponse,
 } from 'src/core/open-api/utils/responses.utils';
 import { getRequestBody } from 'src/core/open-api/utils/request-body.utils';
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
 
 @Injectable()
 export class OpenApiService {
   constructor(
     private readonly tokenService: TokenService,
+    private readonly environmentService: EnvironmentService,
     private readonly objectMetadataService: ObjectMetadataService,
   ) {}
 
   async generateCoreSchema(request: Request): Promise<OpenAPIV3_1.Document> {
-    const schema = baseSchema('core');
+    const baseUrl =
+      this.environmentService.getServerUrl() ||
+      `${request.protocol}://${request.get('host')}`;
```

Comment:
@brendanlaschke To avoid declaration of this 2 times, we can create a private function that compute the `baseUrl`